src/app/pages/token/token-list/token-cw20/token-cw20.component.ts

```diff
@@ -98,6 +115,21 @@ export class TokenCw20Component implements OnInit, OnDestroy {
     this.searchSubject.next(this.textSearch);
   }
 
+  async getListTokenIBC() {
+    this.listTokenIBC = local.getItem(STORAGE_KEYS.LIST_TOKEN_IBC);
+    const res = await this.tokenService.getTokenSupply();
+    const supply = _.get(res, 'data.supply');
+    this.listTokenIBC.forEach((token) => {
+      token.type = ETokenCoinType.IBC;
+      token.isHolderUp = true;
+      supply.forEach((s) => {
+        if (token.denom === s.denom) {
+          token.totalSupply = balanceOf(Number(s?.amount) || 0, +token.decimal) || 0;
```

Comment:
- Không cần chuyển sang number `Number(s?.amount)`
- Sử dụng `getBalance` thay cho `balanceOf` 
```suggestion
          token.totalSupply = getBalance(s?.amount || 0, token.decimal) ;
```